pagopa/io-app
# chore(IT Wallet): [SIW-662] Remove legacy useItwDataProcessing hook

## Short description
Depends on #5241.
This PR removes the legacy `useItwDataProcessing` hook in favour of `useItwInfoBottomSheet` which has the same functionality but generalized for different screens.

## List of changes proposed in this pull request
- Removes `useItwDataProcessing.ts`;
- Replaces its usage with `useItwInfoBottomSheet`.

## How to test
Static checks should be enough but try also a credential issuing flow and check the "Find out more" label in the data presentation screen. It should open a bottom sheet.
